config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -7,13 +7,35 @@ presubmits:
     skip_report: false
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+      - image: gcr.io/k8s-testimages/bootstrap:latest
         args:
         - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
         - "--root=/go/src"
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - --scenario=execute
         - --
         - ./bin/test-on-prow.sh
+        # docker-in-docker needs privileged mode
+        securityContext:
```

Comment:
Given that you're mounting a bunch of system paths and running in a privileged container, can you please provide us with your own kubernetes cluster in which to run these jobs?

Otherwise problems with these jobs runs the risk of harming everyone in the cluster.

Reply:
Can you point me to docs on how we can use our own cluster?

Reply:
1) Create a cluster (`gcloud container clusters create my-cluster` or whatever)
2) Extract the credentials prow needs to schedule pods on your cluster and put them in a secret in prow's cluster (The [prow/cmd/mkbuild-cluster](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/mkbuild-cluster) automates this
  * You'll have to either send this file to test-infra oncall or give them permission to do this.
3) Set the cluster field on your prow jobs:

https://github.com/kubernetes/test-infra/blob/ee541c6392a83f64a1ccd7430de0386c75a1c7e7/config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml#L2-L4